src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -177,27 +176,4 @@ private function decideUnanimous(TokenInterface $token, array $attributes, $obje
 
         return $this->allowIfAllAbstainDecisions;
     }
-
-    /**
-     * TokenInterface vote proxy method.
-     *
-     * Acts as a BC layer when the VoterInterface is not implemented on the voter.
-     *
-     * @deprecated as of 3.4 and will be removed in 4.0. Call the voter directly as the instance will always be a VoterInterface
-     */
-    private function vote($voter, TokenInterface $token, $subject, $attributes)
```

Comment:
`VoterInterface[]` means `array of VoterInterface instances`. So we cannot remove `iterable` from the phpdoc, otherwise we effectively remove the support of iterators from our supported usage (and so from our BC promise). This is quite problematic when our own usage relies on it.